mkdocs.yml

```
@@ -6,12 +1,64 @@
+    - Calibrations: guide/calibrations.md
+  - API Reference:
+    - Core:
+      - Base: api/core/base.md
+      - Elements: api/core/elements.md
+      - Engine: api/core/engine.md
+      - Primitives: api/core/primitives.md
+    - Experiments:
+      - Builtin: api/experiments/builtin.md
+      - Custom: api/experiments/base.md
+    - Theory:
+      - Simulation: api/theory/simulation.md
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)